### ISSUE TITLE
[BUG] Wierd Lore Convertion

### ISSUE THREAD

**Issue by xMystik** [oraxen/oraxen#1534]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I am using ProtocolLib 5.0
- [X] I have checked if any similar bug reports exist

### Description

Basically a previously used setup for the item lores is not working as it should. Although it shows as it has to on the Oraxen Inventory, when the item is actually taken the lore is different. Below is the old setup that was working just fine but now it doesn't.

iron_uncommon:
  template: mining
  material: RAW_IRON
  displayname: <green><bold>Iron <dark_gray>[<gold>✯✯<gray>✯✯✯<dark_gray>]
  lore:
    - "<white> <gray>Rarity <green>Uncommon"
    - "<dark_gray><st>            </st><dark_gray><<dark_aqua>Description<dark_gray>><dark_gray><st>            "
    - "<gray>A mineral found on boulders of"
    - "<gray>the Sparklion!"
    - "<dark_gray><st>                                         "

And below is the new setup which gives the desired result.

iron_common:
  template: mining
  material: RAW_IRON
  displayname: <white><bold>Iron <dark_gray>[<gold>✯<gray>✯✯✯✯<dark_gray>]
  lore:
    - "<white> <gray>Rarity <white>Common"
    - "<st><dark_gray>            </st><dark_gray><<dark_aqua>Description<dark_gray>><st><dark_gray>            </st>"
    - "<gray>A mineral found on boulders of"
    - "<gray>the Sparklion!"
    - "<dark_gray><st>                                         "

### Steps to reproduce

Basically create an item with the given lores as shown above

### Plugin Version

Oraxen version 1.182.0

### ProtocolLib Version

ProtocolLib version 5.3.0-SNAPSHOT-732

### Server Software & Version

git-Paper-463 (MC: 1.20.4)

### How breaking is the bug?

Minor Bug - Plugin completely functional, but features have non-working aspects

**Comment by Boy0000** [comment]
fixed next update
